### ISSUE TITLE
Accessibility: Sign and encrypt buttons do not indicate their state to screen readers

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [paulhenry46/pgp-plugin#1]
On version 1.0.1, the buttons for signing and/or encrypting an email do not indicate whether they are toggled on or off, making it difficult to determine their state.

I haven't yet found a setting to encrypt or sign outgoing emails by deault, so i presume the buttons are always in `off` state upon drafting a new email, but what happens when replying to an inbound encrypted email?

The fix should be simple, in any case: just add `aria-pressed="true|false"` to the buttons in accordance with their state. I'll try to supply a matching PR, though I'm not familiar with Bulwark's plugin system just yet.

**Comment by paulhenry46** [comment]
Hi, thanks for your feedback!

I added the aria properties to these buttons. This fix will be available in the next release, which usually follows a Bulwark update.

As for default options, you can configure them in the plugin settings! In Bulwark, plugins can declare a custom settings page (accessible via the App section) as well as a settings page in the Plugins section. These settings are used when writing an email and responding, but encryption/signing of "Quick reply" is currently not supported.

However, there is currently a bug in Bulwark where default setting values remain undefined initially, even if declared by the plugin. To fix this on your end, simply toggle each setting once in the settings page to save your preferred defaults.

<img width="1920" height="941" alt="Image" src="https://github.com/user-attachments/assets/13adaeb1-3633-45df-9f71-0d99358f9b9f" />
